components/form/Form.tsx

```diff
@@ -141,7 +141,7 @@ export type WrappedFormUtils<V = any> = {
   /** 获取一个输入控件的值 */
   getFieldValue(fieldName: string): any;
   /** 设置一组输入控件的值 */
-  setFieldsValue(obj: Object, callback?: Function): void;
+  setFieldsValue(obj: Partial<V> | any, callback?: Function): void;
```

Comment:
`Partial<V> | any` is same as `any`...

Reply:
Technically yes, but we win the auto completion in the code editor.

Reply:
@ztplz , how do you think about this?

Reply:
A little strange, just for auto completion?

Reply:
Because of the nest path, yes. Better autocompletion is one of the interest of TypeScript no ?

Reply:
@pierre-H  
![image](https://user-images.githubusercontent.com/17755202/61868298-ae94d000-af0b-11e9-9525-c2395d25b42c.png)
`any` here doesn't seem to be necessary because it has a defaulted `any`.
![image](https://user-images.githubusercontent.com/17755202/61868409-e734a980-af0b-11e9-8e0a-5a76db65c65c.png)